src/Views/DriversView.vala

```diff
@@ -75,7 +73,7 @@
         content_area.append (message_box);
 
         var back_button = new Gtk.Button.with_label (_("Back"));
-        back_button.clicked.connect (() => ((Adw.Leaflet) get_parent ()).navigate (BACK));
+        back_button.clicked.connect (() => ((Adw.NavigationView) get_parent ()).pop ());
```

Comment:
Honestly I think the each AbstractInstallerView shouldn't know that its parent is Adw.NavigationView; it should just emit a signal like `previous_step ()`, connect to the signal in MainWindow, and call `Adw.NavigationView.pop ()` in its handler. But that's out of scope of this PR, so leaving just a comment.

Reply:
Made it into an action :)